src/surrogates/src/InterpolationSurrogateIOASCII.C

```diff
@@ -49,7 +49,7 @@ namespace QUESO
                                                  int reading_rank )
   {
     // Root processor
-    unsigned int root = reading_rank;
+    int root = reading_rank;
```

Comment:
Yup.  That should hold us for a decade or more; at least until we need to make the MPI-5 upgrade to handle 2.2 billion rank machines.